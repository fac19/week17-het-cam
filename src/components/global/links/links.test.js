import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Link from "./Links";

test("Link component is styled", () => {
  const tree = renderer.create(<Link />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("color", "#026CB6");
});
