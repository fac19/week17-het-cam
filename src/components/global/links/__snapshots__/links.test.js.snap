// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component is styled 1`] = `
.c0 {
  font-family: "Josefin Sans",sans-serif;
  -webkit-transition: all 0.2s ease-in-out;
  transition: all 0.2s ease-in-out;
  color: #026CB6;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  display: block;
  text-align: center;
}

.c0:hover {
  font-size: 1.2rem;
  color: #E55982;
}

<a
  className="c0"
/>
`;
